### PR TITLE
Backport to branch(3) : Add cross-partition scan options

### PR DESCRIPTION
**Pull request by jnmt**
Backport of #1294

Not just fixing the conflicts, but I changed the default value of the cross-partition scan option to keep backward compatibility in v3.x. Please take a quick look when you get a chance.

### CODE REVIEW

File: core/src/integration-test/java/com/scalar/db/storage/cassandra/CassandraEnv.java [hunk 1]
```diff
@@ -24,6 +24,9 @@ public static Properties getProperties(@SuppressWarnings("unused") String testNa
     props.setProperty(DatabaseConfig.CONTACT_POINTS, contactPoints);
     props.setProperty(DatabaseConfig.USERNAME, username);
     props.setProperty(DatabaseConfig.PASSWORD, password);
+    props.setProperty(DatabaseConfig.CROSS_PARTITION_SCAN, "true");
```
Comment (jnmt) [inline core/src/integration-test/java/com/scalar/db/storage/cassandra/CassandraEnv.java:27]: Changed to `true` in v3.x.

File: core/src/main/java/com/scalar/db/storage/cassandra/CassandraConfig.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.scalar.db.storage.cassandra;
+
+public class CassandraConfig {
```
Comment (jnmt) [inline core/src/main/java/com/scalar/db/storage/cassandra/CassandraConfig.java:3]: In v3.x, there isn't `CassandraConfig`, but I added it to provide `STORAGE_NAME`, as the same as other storages.

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**